QuadradiusR/scripts/network_handler.gd

```diff
@@ -17,11 +17,15 @@ func _ready():
 		url = JavaScript.eval('window.location.href')
 		if not url.ends_with('/'):
 			url = url.rsplit('/', true, 1)[0]
+	if EmbeddedServer.is_available():
```

Comment:
You will overwrite the url set by this callback in the next line (22)
- [ ] fix variable overwrite